k8s/helm/templates/deployment.yaml

```diff
@@ -61,6 +61,8 @@ spec:
               value: /keys/keys.yml
             - name: EQ_RABBITMQ_ENABLED
               value: "False"
+            - name: EQ_ENABLE_HTML_MINIFY
+              value: "False"
```

Comment:
PR says default minification in runner to False, but this is only done on deployed envs atm. `EQ_ENABLE_HTML_MINIFY` in `settings.py` should be changed to `"False"` as the default. Which would mean this change could also be removed as it doesn't override the default.

Reply:
The reason it is disabled is to boost performance for census. If we were to disable it by default, we may as well remove it entirely.

Reply:
The intent was to get it merged, see the performance impact on the daily test, then decide whether to remove it completely in a separate card.

Reply:
Sounds good 👍